AdminCore.WebApp/Properties/launchSettings.json

```diff
@@ -18,11 +18,11 @@
     },
     "AdminCore.WebApp": {
       "commandName": "Project",
-      "launchBrowser": true,
+      "launchBrowser": false,
```

Comment:
why this change?

Reply:
This launches your system default browser at the defined host path when you run the app in an ide i.e localhost:8081

I didn't think this was necessary

Reply:
I know what it does, this just seemed out of context. That's fair enough.